### PR TITLE
Added example documentation for usign rdup.

### LINKED ISSUE #38: add an example incremental backup script
It would be great to have an example incremental backup script somewhere.

### PR DESCRIPTION
**Pull request by phillipCouto**
Closes #38

### CODE REVIEW

File: docs/examples/regular-backups-with-rbup.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Using rdedup with rbup for regular backups
```
Comment (dpc) [inline docs/examples/regular-backups-with-rbup.md]: `rdup` :)

**Comment by phillipCouto** [comment]
@dpc good catch =)

**Comment by dpc** [comment]
Which version of `rdup` are you using? I am using 1.1.15 and there's no `-c` flag there anymore.

**Comment by phillipCouto** [comment]
I am using 1.1.14 lol. I can remove it.

**Comment by phillipCouto** [comment]
@dpc ready for your review.

**Comment by dpc** [comment]
It seems `-c` was re-purposed to mean "force output to tty" as some point (pre 1.0?), and it used to mean "output file body as well", which was quite important if you actually want to take a backup. You made me scared that my backups are not actually taking backups (even though I tested it). :)

**Pull request merged**